source/dub/packagemanager.d

```diff
@@ -445,138 +494,126 @@ class PackageManager {
 		logInfo("Removed package: '"~pack.name~"'");
 	}
 
-	Package addLocalPackage(Path path, string verName, LocalPackageType type)
```

Comment:
The work is now done as member of Repository, but there's a Path-based function here to refer to the right member of m_repos.